Formula/ghostscript.rb

```diff
@@ -57,6 +57,13 @@ class Ghostscript < Formula
     sha256 "0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401"
   end
 
+  if MacOS.version <= :sierra
+    patch do
+      url "https://gitlab.gnome.org/Infrastructure/gimp-macos-build/-/raw/master/patches/ghostscript-9.55.0-leptonica-MacOSX10.12.sdk-no-open_memstream.patch"
+      sha256 "b41981f0f5e9fc2a6666a1992cd1a4ad1fc7a41d9c7d9cd702aaa82716ae5361"
+    end
```

Comment:
It's a third party patch which has been maintained as part of building `gimp` on Mac for 10.12.

Reply:
Has it been sent upstream?

Reply:
I have just done that (should have done it earlier). Here is the reference: https://bugs.ghostscript.com/show_bug.cgi?id=705679. This looks like it might have to go upstream one more step.

Reply:
@carlocab I've now also upstreamed all the way to leptonica https://github.com/DanBloomberg/leptonica/pull/631

Reply:
Great; let's use the patch at https://github.com/DanBloomberg/leptonica/commit/848df62ff7ad06965dd77ac556da1b2878e5e575.patch?full_index=1 instead.

You can use `directory` to specify the directory to apply the patch to.

Reply:
OK, will do.

Reply:
OK, this is done now (and tested, etc).